code/modules/surgery/surgery.dm

```diff
@@ -82,10 +82,15 @@
 		spawn(duration * tool.toolspeed)//in case the player doesn't go all the way through the step (if he moves away, puts the tool away,...)
 			tool.icon_state = "[initial(tool.icon_state)]_off"
 
-	if(digging)
-		playsound(target, 'sound/items/hemostatdig.ogg', 75, 1)
+	if((M_CLUMSY in user.mutations) && prob(20))
+		var/clownsound = null
```

Comment:
Gee I wonder who could have come up with a temp var to hold the sound file to be played in a single playsound proc at the end of an if/else if/else statement yhat would have sorted out the interactions.

Reply:
wyci